src/Compilers/CSharp/Test/Emit3/FirstClassSpanTests.cs

```diff
@@ -8062,7 +8062,121 @@ static class C
             }
             """;
         var comp = CreateCompilationWithSpanAndMemoryExtensions(source);
-        CompileAndVerify(comp, expectedOutput: "2212").VerifyDiagnostics();
+        CompileAndVerify(comp, expectedOutput: "2112").VerifyDiagnostics();
+    }
```

Comment:
>} [](http://example.com/codeflow?start=4&length=1)

Please add a test for the identity conversion between type arguments. Perhaps: `ReadOnlySpan<(int, int)>` and `Span<(int X, int Y)>`, or `ReadOnlySpan<dynamic>` and `Span<object>`.